### ISSUE TITLE
wrong error messages are thrown

### ISSUE THREAD

**Issue by ymark** [isl/x3ml#42]
When the XPATH expression for an instance generator does not exists, the error message reports that it is a label_generator instead of an instance_generator..For example, although the following generator has been declared in the mappings file:
```
<instance_generator name="LocalTermURI">
    <arg name="term" type="xpath">text()</arg>
</instance_generator>   
```
the following is reported:
```
<label_generator name="LocalTermURI">
    <arg name="term" type="xpath">text()</arg>
</label_generator>   
```

**Comment by ymark** [comment]
This happens because we use a common class for modeling both
* instance_generators and 
* label_generators 

This class is eu.delving.x3ml.engine.X3ML.GeneratorElement and has the streamAlias (XStream specific) "label_generator"

**Comment by ymark** [comment]
Fixed (commit 7a9b5c58) by creating an interface with name GeneratorElement, which is implemented by two particular classes:
* eu.delving.x3ml.engine.X3ML.InstanceGeneratorElement
* eu.delving.x3ml.engine.X3ML.LabelGeneratorElement
